tests/autobahn/test_autobahn.py

```diff
@@ -0,0 +1,32 @@
+import os
+import shutil
+
+import pytest
+from python_on_whales import DockerClient
+
+
+@pytest.fixture(scope="session", autouse=True)
+def create_report_directory():
+    path = "reports"
+    if os.path.isdir(path):
+        shutil.rmtree(path)
+    os.mkdir(path)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def build_aiohttp_docker_image():
+    docker = DockerClient(compose_files=["tests/autobahn/docker-compose.yml"])
+    yield docker.compose.build()
+    docker.compose.rm()
```

Comment:
You don't use the result of this fixture so don't yield it.
```suggestion
    docker.compose.build()
    try:
        yield
    finally:
        docker.compose.rm()
```